source/extensions/filters/http/aws_lambda/config.cc

```diff
@@ -32,6 +32,19 @@ getInvocationMode(const envoy::extensions::filters::http::aws_lambda::v3::Config
   PANIC_DUE_TO_CORRUPT_ENUM;
 }
 
+Extensions::Common::Aws::CredentialsProviderSharedPtr
+getCredentialsProvider(const std::string& profile,
```

Comment:
can't we make this work without this function?
we would need to pass in profile string in DefaultCredentialsProviderChain

Reply:
we can, but in case a user decided to use a custom profile, that is because he knows there should be picked from aws credential files, using the complete chain in case the file is not defined will gives an unexpected behaviour, don't you think?